web/dataselectie/datasets/hr/views.py

```diff
@@ -199,3 +202,9 @@ def sanitize_fields(self, item, field_names):
         item.update(
             {field_name: stringify_item_value(item.get(field_name, None))
              for field_name in field_names})
+
+        if item.get('centroid'):
+            lon, lat = item['centroid'].split(' | ')
+            item['lon'] = float(lon)
+            item['lat'] = float(lat)
+        item.pop('centroid')
```

Comment:
Als centroid wordt verwijderd , hoe wordt hij dan weergegeven in de CSV ? Of hij is niet leeg of hij word helemaal niet toegevoegd